### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jailton**
Add instruction to access API reference pages URL locally

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/language en

**Comment by bradtopol** [comment]
This looks like one for @jimangel.
/assign @jimangel

File: README.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ make container-image
 make container-serve
 ```
 
+Open up your browser to http://localhost:1313/docs/reference/kubernetes-api/ to view the API reference pages.
```
Comment (celestehorgan) [inline README.md:103]:
```suggestion
In a web browser, go to http://localhost:1313/docs/reference/kubernetes-api/ to view the API reference.
```

**Review by celestehorgan — CHANGES_REQUESTED** [review]

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**